client/components/FishList.jsx

```diff
@@ -44,8 +42,9 @@ class FishList extends React.Component {
   }
 }
 
-function sortingAZ (items) {
-  items.sort((a, b) => {
+export function sortingAZ (items) {
+  console.log(items)
```

Comment:
Please remove the console.log statement!

Reply:
Done!